index.html

```diff
@@ -486,7 +486,26 @@ <h1>Maxwell's equations in vacuum</h1>
             </figcaption>
           </figure>
         </section>
+
+        <section>
+          <h1>Copyrights</h1>
+          <p>This section describes how place the copyright notice at the bottom of the document. You can add a copyright notice through a paragraph <code>&lt;p&gt;</code> within a <code>&lt;footer&gt;</code> element. Note that <code>&lt;footer&gt;</code> has to be the last child of element <code>&lt;main&gt;</code>.</p>
+          <figure>
+            <pre><code>&lt;main&gt;
+  ...
+  &lt;footer&gt;
+    &lt;p&gt;&lt;small&gt;...&lt;/small&gt;&lt;/p&gt;
+  &lt;/footer&gt;
+&lt;/main&gt;</code></pre>
+            <figcaption>
+              <p>Code snippet for stating content attribution or licensing.</p>
+            </figcaption>
+          </figure>
+        </section>
       </article>
+      <footer>
+        <p><small>Created with &lt;3 in <a href="https://open.nextbit.it/">Nextbit</a> | <a href="https://github.com/nextbitlabs/Rapido/blob/master/LICENSE">MIT License</a></small></p>
```

Comment:
Can we use `❤`? It's UTF-8

Reply:
I am not convinced. The issue with the UTF-8 heart is that it looks different in different devices. For example, in iPhone (Safari) it is coloured with a shade of reds and this brokes the style.

Reply:
Ah I see, then I agree with you!